config.go

```diff
@@ -33,6 +36,17 @@ type Process struct {
 	Cwd string `json:"cwd"`
 }
 
+// User specifies linux specific user and group information for the container's
+// main process.
+type User struct {
```

Comment:
No, because the intention is that as other platforms have their own fields, they still fall in here.